common/android_helpers.h

```diff
@@ -1,5 +1,5 @@
 // License: Apache 2.0. See LICENSE file in root directory.
-// Copyright(c) 2017 Intel Corporation. All Rights Reserved.
+// Copyright(c) 2019 Intel Corporation. All Rights Reserved.
```

Comment:
According to this ([link](https://www.copyrightlaws.com/copyright-symbol-notice-year/)) the year marks when the work was first published.
It is not really a rule, more a convention to keep the original date. But re-tagging the files, imho will produce extra-burden both for PR submitters and reviewers.